src/main/java/com/example/indiproject/services/impl/AppUserServiceImpl.java

```diff
@@ -32,12 +33,16 @@ public boolean appUserWithUserNameExists(String userName) {
     public boolean appUserNameIsAppropriate(String userName) {
         userName = userName.toLowerCase();
         if (userName.contains("fuck")) {
+            log.error("Forbidden name: contains: \"fuck\"");
```

Comment:
Neni to uplne soucasti ukolu, ale napadlo me ze by bylo "hezci" to trochu zrefactorovat aby ta oskliva slovicka byla v nejakem seznamu stranou, pak by ta hlaska mohla byt jen na jednom radku (s doplnenim o ktere zakazane slovo slo)